hack/generator/pkg/jsonast/swagger_type_extractor.go

```diff
@@ -65,15 +68,15 @@ func (extractor *SwaggerTypeExtractor) ExtractTypes(
 			continue
 		}
 
-		resourceSchema := extractor.findARMResourceSchema(filePath, swagger, *put)
+		resourceSchema := extractor.findARMResourceSchema(filePath, *put)
 		if resourceSchema == nil {
-			//klog.Warningf("No ARM schema found for %s in %q", rawOperationPath, filePath)
+			// klog.Warningf("No ARM schema found for %s in %q", rawOperationPath, filePath)
```

Comment:
Should this warning be commented out, or no?

Reply:
Will address in a future PR; change here was formatter running